.eslintrc

```diff
@@ -14,14 +14,15 @@
       "number": true,
       "string": true
     }],
-    "no-unused-expressions": [2, {
+    "no-unused-expressions": [0, {
```

Comment:
I'd generally say this is a bad sign. What was the use case for this and could it be solved with an inline comment rather than a blanket rule change